addons/l10n_it_edi/views/l10n_it_view.xml

```diff
@@ -137,6 +137,12 @@
                         invisible="l10n_it_edi_state not in ('being_sent', 'processing', 'forward_attempt')"
                     />
                 </xpath>
+                <xpath expr="//button[@name='action_invoice_sent']" position="inside">
+                    <!--Explicitly add the proxy EDI mode to the Send button label when the mode is in demo/test (e.g. "Send (Demo)")-->
+                    <span>Send</span>
+                    <span invisible="l10n_it_edi_proxy_mode not in (False, 'demo')"> (Demo)</span>
+                    <span invisible="l10n_it_edi_proxy_mode != 'test'"> (Test)</span>
```

Comment:
As discussed, should only be visible if the user is actually sending through the EDI.